ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCleanup/FSharpReformatCode.fs

```diff
@@ -79,8 +79,8 @@ type FSharpReformatCode(textControlManager: ITextControlManager) =
                     sourceFile.GetPsiServices().Files.CommitAllDocuments()
                 with _ -> ()
             else
-                let textControl = textControlManager.VisibleTextControls |> Seq.find (fun c -> c.Document == document)
-                let cursorPosition = textControl.Caret.Position.Value.ToDocLineColumn()
+                let textControl = textControlManager.VisibleTextControls |> Seq.tryFind (fun c -> c.Document == document)
```

Comment:
What if there are multiple editors with the same document? This check seems to ignore which editor was actually focused before the reformat. Please try to use data contexts or other ways to get the proper text control.